packages/azure-openapi-validator/autorest/src/spectral-plugin-func.ts

```diff
@@ -36,7 +36,8 @@ export async function spectralPluginFunc(initiator: IAutoRestPluginInitiator): P
   }
 
   const resolvedOpenapiType: OpenApiTypes = getOpenapiType(openapiType)
-  const [ruleset, namesOfRulesInStagingOnly]: [Ruleset, string[]] = await getRuleSet(resolvedOpenapiType)
+  // FIXME: This should be set by the CI, not hardcoded
```

Comment:
I think some component should look for the `x-typespec-generated` flag in swagger to decide whether to enable this.

Reply:
Can you please elaborate (in code or comment) what is the `this` here?